source/shared/cpp/ObjectModel/CollectionTypeElement.h

```diff
@@ -111,9 +116,12 @@ namespace AdaptiveSharedNamespace
 
         // Parse Items
         collection->DeserializeChildren(context, value);
+        
```

Comment:
Nit: why the extra newlines

Reply:
thanks!

---
In reply to: [275094349](https://github.com/Microsoft/AdaptiveCards/pull/2675#discussion_r275094349) [](ancestors = 275094349)